enclave/core/sgx/report.c

```diff
@@ -127,14 +202,21 @@ static oe_result_t _get_sgx_target_info(
     size_t opt_params_size,
     sgx_target_info_t* target_info)
 {
+    oe_result_t result = OE_UNEXPECTED;
     uint32_t retval;
 
-    if (oe_get_qetarget_info_ocall(
-            &retval, format_id, opt_params, opt_params_size, target_info) !=
-        OE_OK)
-        return OE_FAILURE;
+    OE_CHECK(oe_get_qetarget_info_ocall(
+        &retval, format_id, opt_params, opt_params_size, target_info));
+    result = (oe_result_t)retval;
 
-    return (oe_result_t)retval;
+done:
+    if (result == OE_UNSUPPORTED)
+        OE_TRACE_WARNING(
+            "sgx remote attestation is not fully supported. To "
```

Comment:
should this say "SGX remote attestation is not enabled." instead of "fully supported"?  And same question on line 253 and in other files.  In other words, if OE_UNSUPPORTED is returned above, is there still some part of SGX remote attestation that is partially supported without adding the import in the EDL file?

Reply:
Only one call (`oe_verify_local_report_ecall`), which is part of verifier, can be partially enabled. The rest calls are part of attester so should be enabled all together. Given that, I update the log message as you suggested and leave the message of `oe_verify_local_report_ecall` as the same.